src/components/MessageDisplay.svelte

```diff
@@ -100,11 +101,15 @@
   export let selectedItems = [];
 
   const banMessage = item => () => {
-    channelFilters.set(item.authorId, {
-      ...channelFilters.get(item.authorId),
-      name: item.author,
-      blacklist: true,
-    });
+    if (item.types & AuthorType.mchad) {
+      mchadUsers.set(item.author, true);
```

Comment:
`mchadUsers` is undefined.

Reply:
My bad, I think I had set `window.mchadUsers` and then forgot about it when I removed it.